daemonset.yaml

```diff
@@ -44,39 +39,36 @@ spec:
       - name: dev
         hostPath:
           path: /dev
-      - name: nvidia-overlay
+      - name: nvidia-install-dir-host
         hostPath:
           path: /home/kubernetes/bin/nvidia
-      - name: os-release
-        hostPath:
-          path: /etc/os-release
-      - name: sysrq
+      - name: root-mount
         hostPath:
-          path: /proc/sysrq-trigger
+          path: /
       initContainers:
-      - image: "gcr.io/google-containers/cos-nvidia-driver-install@sha256:de2d9098df423b064bb2579c4fb114022b2a6b7cc39c677d7ac720c9b75d9eb5"
-        command: ["/bin/sh", "-c"]
-        args: ["usr/bin/nvidia-installer.sh"]
+      - image: gcr.io/cos-cloud/cos-gpu-installer:v20171215
         name: nvidia-driver-installer
         resources:
           requests:
             cpu: 0.15
         securityContext:
           privileged: true
         env:
-          - name: BASE_DIR
-            value: "/rootfs/nvidia"
-          - name: DEVICE_PLUGIN_ENABLED
-            value: "true"
+          - name: NVIDIA_DRIVER_VERSION
+            value: "384.81"
+          - name: NVIDIA_INSTALL_DIR_HOST
+            value: /home/kubernetes/bin/nvidia
```

Comment:
Din't we decide to change this to `/var/lib/nvidia` or something like that? @Amey-D

Reply:
We can set this to `/var/lib/nvidia` but then I have to update the device-plugin as well.

Reply:
Also, I think that requires some more hoops: https://github.com/GoogleCloudPlatform/cos-gpu-installer/blob/master/install-test-gpu.cfg#L41